src/jit/importer.cpp

```diff
@@ -3458,6 +3458,21 @@ GenTree* Compiler::impIntrinsic(GenTree*                newobjThis,
             ni = lookupNamedIntrinsic(method);
 
 #ifdef FEATURE_HW_INTRINSICS
+            if (ni == NI_IsSupported_True)
```

Comment:
We might be able to make this more general and move it out of `FEATURE_HW_INTRINSICS`; so we have a single spot for methods which are statically true/false...

Reply:
A few are probably: `BitConverter.IsLittleEndian`, `Environment.Is64BitProcess`, and `Vector.IsHardwareAccelerated` which rely on other explicit recognition or JIT tricks.

There are likely others already and more to exist in the future as well.... The `return gtNewIconNode(true);` code isn't particular complex; but it might be nice to just be able to do the check and return `NI_IsSupported_True` or `NI_IsSupported_False` from `lookupNamedIntrinsic`; rather than having multiple separate paths that all do the same thing.